src/pheval_exomiser/prepare/create_batch_commands.py

```diff
@@ -131,75 +157,140 @@ def write_command(
-                + str("/exomiser-testdata-phenopacket/" + command_arguments.sample.name)
-                + " --vcf "
-                + str("/exomiser-testdata-vcf/" + command_arguments.vcf_file.name)
-                + " --assembly "
-                + command_arguments.vcf_assembly
-                + "\n"
-            )
+                " --output-directory " + str(command_arguments.raw_results_dir)
+            ) if command_arguments.raw_results_dir is not None else None
+        except IOError:
+            print("Error writing ", self.file)
+
+    def write_output_options(self, command_arguments: ExomiserCommandLineArguments) -> None:
+        """Write a command out for exomiser ≤13.1.0 to run - including output option file specified."""
+        try:
+            self.file.write(
+                " --output " + str(command_arguments.output_options_file)
+            ) if command_arguments.output_options_file is not None else None
         except IOError:
             print("Error writing ", self.file)
 
-    def write_command_output_options(
-        self, analysis_yaml, command_arguments: ExomiserCommandLineArguments
+    def write_analysis_command(
+        self, analysis_yaml: Path, command_arguments: ExomiserCommandLineArguments
+    ):
+        self.write_basic_analysis_command(analysis_yaml, command_arguments)
+        self.write_results_dir(command_arguments)
+        self.write_output_options(command_arguments)
+        self.file.write("\n")
+
+    def write_basic_phenotype_only_command(
+        self, command_arguments: ExomiserCommandLineArguments
     ) -> None:
-        """Write a command out for exomiser to run - including output option file specified."""
+        """Write a phenotype-only command out for exomiser ≥13.2.0 to run."""
         try:
             self.file.write(
-                "--analysis "
-                + str(analysis_yaml)
-                + " --sample "
+                "--sample "
                 + str(command_arguments.sample)
-                + " --vcf "
-                + str(command_arguments.vcf_file)
-                + " --assembly "
-                + command_arguments.vcf_assembly
-                + " --output "
-                + str(command_arguments.output_options_file)
-                + "\n"
+                + " --output-directory "
+                + str(command_arguments.raw_results_dir)
+                + " --output-filename "
+                + f"{command_arguments.sample.stem}-exomiser"
+                + " --preset "
+                + "phenotype-only"
             )
         except IOError:
             print("Error writing ", self.file)
 
-    def write_docker_command_output_options(
-        self, analysis_yaml, command_arguments: ExomiserCommandLineArguments
-    ) -> None:
-        """Write a docker command out for exomiser to run - including output option file specified."""
+    def write_phenotype_only_command(self, command_arguments: ExomiserCommandLineArguments):
```

Comment:
All the write_thing and write_docker_thing methods are identical! The output is also identical (i.e. it's an Exomiser CLI string). Just do it once and pass in the relevant paths.